### DISCUSSION TITLE
Live connection count 2/2

### DISCUSSION THREAD

**Discussion by JiaOkay** [dbeaver/dbeaver#11998] [Q&A]
#### System information: 
- Windows 10 专业版  20H2
- DBeaver 21.0.0
- Additional extensions

#### Connection specification:
- mysql 5.6
- jdbc
- No tunnels or proxies

#### 1、我只开了一个脚本窗口，然后我在左侧打开数据库，展开查看表之后，执行查询语句时会提示有两个活跃的连接。
 关闭连接后重新连接则恢复正常，之前的版本也有这个问题。
![image](https://user-images.githubusercontent.com/38833358/110060482-f4c93b00-7da0-11eb-89d5-62f7d34db4d3.png)

#### 2、同时执行两句，用分号";"隔开，并不能返回两个结果集。会提示SQL错误，无论是Oracle 还是MySQL 都一样。
![image](https://user-images.githubusercontent.com/38833358/110063461-285a9400-7da6-11eb-832b-ec2a8963687d.png)

**Comment by uslss** [comment]
hi @JiashunLi ,
1. Probably Open separate connection for each editor option is enabled
![image](https://user-images.githubusercontent.com/31996417/110535135-f09a7600-8130-11eb-91f0-8a522129d1fe.png)
2. Try to use Execute SQL script button (Alt+x)
![image](https://user-images.githubusercontent.com/31996417/110535358-39eac580-8131-11eb-947c-30f0638b2326.png)
